client/apps/eternum-mobile/src/pages/realm/ui/tabs/military-tab.tsx

```
@@ -32,7 +32,7 @@ const getDangerLevel = (distance: number) => {
   };
 };
 
-export function MilitaryTab({ }: MilitaryTabProps) {
+export function MilitaryTab({}: MilitaryTabProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)